content/pages/volunteers.md

```diff
@@ -0,0 +1,96 @@
+Title: Volunteer Opportunities
+Category: pages
+Summary: Volunteer opportunities available with WiCS undergraduate.
+
```

Comment:
Not really much to fix there. We can't change the default on that.

I believe that we do not want category or summary metadata here since those wouldn't ever get exposed anyways.